pkg/controlsvc/reload.go

```diff
@@ -16,34 +17,29 @@ type (
 	reloadCommand     struct{}
 )
 
-var configPath = ""
+var (
+	configPath       = ""
+	mu               sync.Mutex
+	cfgPrevious      = make(map[string]struct{})
```

Comment:
Why are configuration items empty structs?

Reply:
Sorry I was unclear. The type definition of the map value is a struct with no fields. Why is that not a concrete type ... or `interface{}` if it needs to be purely dynamic?